juju_verify/juju_verify.py

```diff
@@ -121,6 +122,9 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument('-l', '--log-level', type=str.lower,
                         help='Set amount of displayed information',
                         default='info', choices=['trace', 'debug', 'info'])
+    parser.add_argument("--stop-on-failure", action="store_true",
```

Comment:
Should we come up with a short version of this? `-f` is generally related to `--force` but we could use sit as `fast-fail`. Alternatively, `-s` from `stop` ?

Reply:
I don't like using `-f`, because it usually represents `--force` or `--format`. Using `-s` sound good to me.